launcher/src/components/UI/server-management/components/server-update/UpdateServer.vue

```diff
@@ -98,15 +108,37 @@ const stereumApp = ref({
 const numberOfUpdatablePackages = ref(null);
 const searchingForUpdatablePackages = ref(false);
 
+console.log("getAllTasks", serverStore.allTasks);
 const newUpdates = computed(() => serverStore.upgradablePackages);
 const onOff = computed(() => (stereumApp.value.autoUpdate == "on" ? "text-green-700" : "text-red-700"));
 
+//Watchers
+
+watchEffect(() => {
```

Comment:
i think `watchEffect` is a bit overkill here. you can also use `computed`

Reply:
So from the [docs](https://vuejs.org/guide/essentials/watchers.html#watcheffect) it clearly shows that watchers are used for refs that change on a regular basis, so it can adapt on chages immediately. In this case `osVersionCurrent` will change twice a year at max. Users might not even upgrade at all.